.github/workflows/continuous-integration.yml

```
@@ -81,6 +90,7 @@ jobs:
 
       - name: "Set minimum-stability to stable in Composer"
         run: "composer config minimum-stability stable"
+        if: "${{ matrix.stability == 'stable' }}"
```

```suggestion
        run: "composer config minimum-stability ${{ matrix.stability }}"
```

Let's not rely on what's in the `composer.json`. In fact, I'd change that to `stable` there because it was only introduced in to pull ODM v2.3-dev.

---

👍 (it was introduced [when testing Symfony 5 by the way](https://github.com/doctrine/DoctrineMongoDBBundle/pull/602))